client/analytics/report/downloads/table.js

```diff
@@ -86,10 +89,10 @@ export default class CouponsReportTable extends Component {
 				{
 					display: (
 						<Link href={ productLink } type="wc-admin">
-							{ product_id }
+							{ productName }
 						</Link>
 					),
-					value: product_id,
+					value: productName,
 				},
 				{
 					display: file_name,
```

Comment:
I see this is linked to the file in the designs.  Do we also want to link this here or is that too much overhead?

Reply:
Right, I think that's not possible with the data we currently get from the API without making another request.

I created a follow-up issue, so we can track that there: #1245.